data/mods/Graphical_Overmap/fontlist.txt

```diff
@@ -0,0 +1,2796 @@
+Map Font LARWICK
```

Comment:
You don't need to add this file to a mod - it is generated on end systems automatically.

Reply:
Ahah, that makes a lot of sense, thanks!